packages/create-sitecore-jss/src/templates/node-headless-ssr-proxy/config.js

```diff
@@ -0,0 +1,152 @@
+const fs = require('fs');
+const { RestDictionaryService } = require('@sitecore-jss/sitecore-jss');
+const httpAgents = require("./httpAgents");
+
+/**
+ * The JSS application name defaults to providing part of the bundle path as well as the dictionary service endpoint.
+ * If not passed as an environment variable or set here, any application name exported from the bundle will be used instead.
+ */
+let appName = process.env.SITECORE_JSS_APP_NAME;
```

Comment:
```suggestion
let appName = process.env.SITECORE_JSS_APP_NAME || <%- appName ? appName : '' %>;
```
Do we want ejs here for the appName? Same for the `package.json`

Reply:
See PR description (deciding to tackle this in future work).

I actually started going down this path but then realized the `appName` here is a bit different than the `appName` we're collecting in client apps (Next.js, RAV). This and the ssr-edge samples are server apps, and the `appName` actually refers to the corresponding _client_.